### PR TITLE
style: Get emoji from github api

### PR DESCRIPTION
**Pull request by sy-records**
**Summary**

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [x] Code style update
- [ ] Refactor
- [ ] Docs
- [ ] Build-related changes
- [ ] Other, please describe:

If changing the UI of default theme, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

You have tested in the following browsers: (Providing a detailed version will be better.)

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] IE

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature
- [ ] Related documents have been updated
- [ ] Related tests have been updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

---

* [ ] DO NOT include files inside `lib` directory.

### CODE REVIEW

**Comment by sy-records** [comment]
Yeah, I'll modify it later. 

I'm not sure how to use it. What do you think?

use key => value to get the src address directly, or use JavaScript to get the unicode value?

**Comment by anikethsaha** [comment]
I am not sure about github's api about emoji.

is there any bug with the current setup ?

**Comment by sy-records** [comment]
No, but now the effect is the same as without plugin.

It is not a complete emoji. I will modify it later.

**Comment by anikethsaha** [comment]
ok cool , you can convert this PR to draft and when ready , mark this as ready to review if its ok

File: src/plugins/emoji.js [hunk 1]
```diff
@@ -1,1523 +1,1812 @@
...
+  artificial_satellite: '1f6f0',
+  artist: '1f9d1-1f3a8',
+  aruba: '1f1e6-1f1fc',
+  ascension_island: '1f1e6-1f1e8',
+  asterisk: '002a-20e3',
+  astonished: '1f632',
+  astronaut: '1f9d1-1f680',
+  athletic_shoe: '1f45f',
+  atm: '1f3e7',
+  atom: 'https://github.githubassets.com/images/icons/emoji/atom',
```
Comment (anikethsaha) [inline src/plugins/emoji.js:77]: unicode for this seems missing ?

File: src/plugins/emoji.js [hunk 2]
```diff
@@ -1,1523 +1,1812 @@
...
+  bouncing_ball_woman: '26f9-2640',
+  bouquet: '1f490',
+  bouvet_island: '1f1e7-1f1fb',
+  bow: '1f647',
+  bow_and_arrow: '1f3f9',
+  bowing_man: '1f647-2642',
+  bowing_woman: '1f647-2640',
+  bowl_with_spoon: '1f963',
+  bowling: '1f3b3',
+  bowtie: 'https://github.githubassets.com/images/icons/emoji/bowtie',
```
Comment (anikethsaha) [inline src/plugins/emoji.js:199]: ditto

File: src/plugins/emoji.js (hunk 1 above)
Comment (sy-records) [reply src/plugins/emoji.js]: Yeah, I missed it. 😅

File: src/plugins/emoji.js [hunk 3]
```diff
@@ -1,1523 +1,1812 @@
...
-// https://github.com/scotch-io/All-Github-Emoji-Icons
+// Emoji from GitHub API
+// https://api.github.com/emojis
 window.emojify = function(match, $1) {
-  return AllGithubEmoji.indexOf($1) === -1
+  return AllGithubEmoji.hasOwnProperty($1) === false
     ? match
     : '<img class="emoji" src="https://github.githubassets.com/images/icons/emoji/' +
-        $1 +
+        AllGithubEmoji[$1] +
```
Comment (trusktr) [inline src/plugins/emoji.js:1809]: LGTM.

What's the main difference between the two types of links? f.e.

- https://github.githubassets.com/images/icons/emoji/unicode/1f4a4.png
- https://github.githubassets.com/images/icons/emoji/zzz.png

**Comment by trusktr** [comment]
@anikethsaha @sy-records I'm look at the Now deploy, https://docsify-preview-git-fork-sy-records-style-emoji.docsify-core.now.sh/#/, and if you inspect the :heart:  it seems to use the old URL. Is the Now deploy not working? I thought it would have the new-style URL?

**Comment by sy-records** [comment]
Because there is no emoji plugin loaded.

**Comment by trusktr** [comment]
I haven't tried it: what's the difference between with and without the emoji plugin?

File: src/plugins/emoji.js (hunk 3 above)
Comment (sy-records) [reply src/plugins/emoji.js:1809]: The picture presents different sizes, It should be no problem to use as an emoticon.

**Comment by sy-records** [comment]
If not used, some emoji will not be displayed normally, for example :label:

File: src/plugins/emoji.js (hunk 3 above)
Comment (trusktr) [reply src/plugins/emoji.js:1809]: Gotcha. So smaller images, better for in text.

**Comment by sy-records** [comment]
@anikethsaha OK, I added.

**Pull request merged**